### ISSUE TITLE
Keepass2 Android internal keyboard not showing up

### ISSUE THREAD

**Issue by Bnwqo** [PhilippC/keepass2android#1177]
I recently switched to a new phone and as the title says the internal K2A keyboard isn't popping up / can't switch to it and I want to feel comfortable entering my password when opening my DB file.

Does anyone know how to manually bring it up or enable it?

**Comment by yaakovfeldman** [comment]
https://play.google.com/store/apps/details?id=keepass2android.plugin.keyboardswap2

Install the app and give it the described permissions

**Comment by PhilippC** [comment]
@Bnwqo did you enable the keyboard in the android settings?

**Comment by Bnwqo** [comment]
> @Bnwqo did you enable the keyboard in the android settings?

Yes, it's enabled under "Manage keyboards." I also get the adb steps, test confirguration says, "No permission to change input method."

To change system settings it's allowed. That's what I don't understand, on my old phone before switching I didn't get this issue. Is it the adb requirement then?
